cmd/module.go

```diff
@@ -2,6 +2,7 @@ package commands
 
 import (
 	"os"
+	"sigs.k8s.io/yaml"
```

Comment:
Don't we use `"gopkg.in/yaml.v2"` in the `mtad-gen` command?

Reply:
changed